salt/modules/zypperpkg.py

```diff
@@ -405,7 +405,9 @@ def _get_available_versions(self):
         Get available versions of the package.
         :return:
         '''
-        solvables = self.zypper.nolock.xml.call('se', '-xv', self.name).getElementsByTagName('solvable')
+        solvables = self.zypper.nolock.xml.call(
+            "se", "-xv", self.name, success_retcodes=[104]
```

Comment:
You already calling XML, do you still need "x" option?

Reply:
Seems that is not needed. Removed!